Policies/github-privacy-statement.md

```diff
@@ -197,35 +199,38 @@ Please note that you cannot opt out of receiving important communications from u
+
+
+For Enterprise Marketing Pages, we may also use non-essential cookies to  (i) gather information about enterprise users’ interests and online activities to personalize their experiences, including by making the ads, content, recommendations, and marketing seen or received more relevant and (ii) serve and measure the effectiveness of targeted advertising and other marketing efforts. If you disable the non-essential cookies on the Enterprise Marketing Pages, the ads, content, and marketing you see may be less relevant. We provide more information about [cookies on GitHub](/github/site-policy/github-subprocessors-and-cookies#cookies-on-github) on our [GitHub Subprocessors and Cookies](/github/site-policy/github-subprocessors-and-cookies) page that describes the cookies we set, the needs we have for those cookies, and the expiration of such cookies.
 
 Our emails to users may contain a pixel tag, which is a small, clear image that can tell us whether or not you have opened an email and what your IP address is. We use this pixel tag to make our email communications more effective and to make sure we are not sending you unwanted email.
 
 ### DNT
 
-"[Do Not Track](https://www.eff.org/issues/do-not-track)" (DNT) is a privacy preference you can set in your browser if you do not want online services to collect and share certain kinds of information about your online activity from third party tracking services. GitHub responds to browser DNT signals and follows the [W3C standard for responding to DNT signals](https://www.w3.org/TR/tracking-dnt/). If you would like to set your browser to signal that you would not like to be tracked, please check your browser's documentation for how to enable that signal. There are also good applications that block online tracking, such as [Privacy Badger](https://privacybadger.org/).
+"[Do Not Track](https://www.eff.org/issues/do-not-track)" (DNT) is a privacy preference you can set in your browser if you do not want online services to collect and share certain kinds of information about your online activity from third party tracking services. Some services may respond to browser DNT signals and follow the [W3C standard for responding to DNT signals](https://www.w3.org/TR/tracking-dnt/). If you would like to set your browser to signal that you would not like to be tracked, please check your browser's documentation for how to enable that signal. There are also good applications that block online tracking, such as [Privacy Badger](https://privacybadger.org/) or [uBlock Origin](https://github.com/gorhill/uBlock/).
```

Comment:
Let me prefix this by stating that I am a complete layman.

Previously: \*GitHub\* responds to browser DNT signals and follows the W3C spec.
Now: Some random services, somewhere in the world, hosted by GitHub or somebody else \*may\* respond to browser DNT signals and follow the W3C spec.

Doesn't this change invalidate the whole paragraph and turns it into a generic wiki article?

Reply:
Dunno, they will stop respecting DNT but leave this paragraph and make it seem as if they do. This is just confusing.

Reply:
"Confusing" is one way to put it.

Reply:
:))

Reply:
So; let's get this straight:

1. According to GDPR article 22 data subjects may exercise their right to object to processing using technical specifications.
1. GitHub acknowledges the DNT signal as a valid technical standard, i.e. technical specification.
3. Moreover; GitHub honors - or at least used to honor - that signal, illustrating that they have the capacity to respond to it appropriately.

Yeah... uhm..
How is attempting to weasel yourself out from under that not morally blackest evil?

Reply:
555

Reply:
555

Reply:
555

Reply:
can someone please explain the policy's @github and what happened to @dontsellmyDATA?
asking for a friend